### DISCUSSION TITLE
Who staged the most-attended free admission music concert in history?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#773] [Galaxy Trivia] [answered]
**Bonus:** Where was it held and how many people attended?

**Comment by peterbaccaro** [comment]
The most-attended free admission music concert in history was staged by the French musician **Jean-Michel Jarre** on September 6, 1997, in Moscow, Russia. The concert was held at the **State University of Moscow** to celebrate the city's 850th anniversary, and **an estimated 3.5 million people attended**, which makes it the largest concert ever performed to a live audience.

The concert was titled "The Oxygène Tour" and was part of Jarre's world tour. He performed on a specially constructed stage that was over 100 meters (328 feet) long and used a vast array of pyrotechnics, lasers, and projections to create a spectacular visual show. The concert featured several of Jarre's most popular tracks, including "Oxygène", "Equinoxe", and "Rendez-vous". The event was also broadcast live on television and radio to millions of people around the world.

**Comment by SANKARAMDAS** [comment, marked as answer]
The record for the biggest concert attendance is held by Vasco Rossi (left) for a ticketed concert and Jean-Michel Jarre and Rod Stewart (right) for a free concert.
Both Jean-Michel Jarre's concert in Moscow in 1997 and Rod Stewart's concert in Copacabana in 1994 have been reported to attract audiences of more than 3.5 million people.

**Reply by ParanoidUser** [reply]
Well... in fact, there were two free concerts of this size ([source](https://en.wikipedia.org/wiki/List_of_highest-attended_concerts#Free_concerts)).
